heudiconv/dicoms.py

```diff
@@ -265,8 +265,13 @@ def group_dicoms_into_seqinfos(files, grouping, file_filter=None,
         series_id = '-'.join(map(str, series_id))
         if mw.image_shape is None:
             # this whole thing has no image data (maybe just PSg DICOMs)
-            # nothing to see here, just move on
-            continue
+            # If this is a Siemens PhysioLog, keep it:
+            if mw.dcm_data.SeriesDescription.endswith('_PhysioLog'):
+                # just give it a dummy shape, so that we can continue:
+                mw.image_shape=(0,0,0)
```

Comment:
nit pep8: spaces around ` = `

Reply:
Fixed in 9de5114535f08c3dd3d8a177326a920127d1262c